bentoml/cli/click_utils.py

```diff
@@ -95,10 +146,41 @@ def wrapper(quiet, verbose, *args, **kwargs):
 
         return wrapper
 
+    @staticmethod
+    def bentoml_track_usage(func):
+        @functools.wraps(func)
+        def wrapper(*args, **kwargs):
+            start_time = time.time()
+            try:
+                return_value = func(*args, **kwargs)
+            except BentoMLException as e:
+                duration = time.time() - start_time
+                additional_info = {
+                    'duration': duration,
+                    'error_type': type(e).__name__,
+                }
+                _send_track_info(func.__name__, func.__module__, additional_info)
+                raise ClickException(str(e))
+            except BaseException as e:
+                duration = time.time() - start_time
+                additional_info = {
+                    'duration': duration,
+                    'error_type': type(e).__name__,
+                }
+                _send_track_info(func.__name__, func.__module__, additional_info)
+                raise
+            duration = time.time() - start_time
+            _send_track_info(func.__name__, func.__module__, {"duration": duration})
```

Comment:
let's not use func.__name__, func.__module__, can you find a better way to get the CLI command used?